pyproject.toml

```diff
@@ -75,14 +69,11 @@ include-package-data = true
 include = ["derotation*"]
 exclude = ["tests", "docs*"]
 
-
 [tool.pytest.ini_options]
-addopts = "--cov=derotation"
-
-[tool.black]
-target-version = ['py38', 'py39', 'py310']
-skip-string-normalization = false
-line-length = 79
+addopts = "--cov={{cookiecutter.module_name}}"
```

Comment:
I think this needs to be updated with `derotation`

Reply:
ohh nice catch!!!

Reply:
```suggestion
addopts = "--cov=derotation"
```